admin/webpack.admin.config.js

```diff
@@ -0,0 +1,52 @@
+var webpack = require('webpack');
+var path = require('path');
+var common = require('../webpack.common.config.js');
+var assign = require('object-assign');
+var BundleTracker = require('webpack-bundle-tracker');
+
+var websiteRoot = path.join(__dirname, 'websiteRoot', 'static');
+
+var adminRoot = path.join(__dirname, 'static');
+
+var staticWebsitePath = function(dir) {
+    return path.join(websiteRoot, dir);
+};
+
+var staticAdminPath = function(dir) {
+    return path.join(adminRoot, dir);
+};
+
+common.entry['admin-base-page'] = staticAdminPath('js/pages/base-page.js')
+
+// Adding bundle tracker to plugins
+common.plugins = [
```

Comment:
Note that the above returns a new array; it doesn't modify `common.plugins` in-place.